test/setup.js.travis

```diff
@@ -12,7 +12,10 @@ global.chromeBinaryPath = '/usr/bin/google-chrome-stable';
 global.screenshotDir = 'travis-screenshots';
 
 // URL for local bundle.js
-global.bundleUrl = 'https://localhost.daplie.me:8080/build/bundle.js';
+global.userName = 'freeCodeCamp';
+global.branch = 'master';
+global.bundleUrl = `https://rawgit.com/
+  ${global.userName}/testable-projects-fcc/${global.branch}/build/bundle.js`;
```

Comment:
... so if I were submitting a new PR, for example, `fix/thing-to-fix`, I would push fixes to my fork before running tests, then change these variables to `global.userName = 'tbushman', global.branch = 'fix/thing-to-fix'`.